batched/dense/impl/KokkosBatched_HostLevel_Gemm_Serial_Impl.hpp

```diff
@@ -153,8 +153,22 @@ class BatchedSerialGemm {
     // matrix transpositions, here we must perform the GEMM on:
     // row_vec x col_vec, which is svA_row' x svB_col to compute the element
     // of C.
-    KokkosBatched::SerialGemm<Trans::Transpose, Trans::NoTranspose, ArgMode>::invoke(alpha, svA_row, svB_col, beta,
-                                                                                     svC_ele);
+    // KokkosBatched::SerialGemm<Trans::Transpose, Trans::NoTranspose, ArgMode>::invoke(alpha, svA_row, svB_col, beta,
+    //                                                                                    svC_ele);
+    using ValueType             = typename CViewType::value_type;
+    ValueType svA_row_x_svB_col = 0;
+    // KokkosBatched::SerialDotInternal::invoke(svA_row.extent(0), svA_row.data(), svA_row.stride(0),
```

Comment:
This didn't work?

Reply:
No I couldn't make it work.

Reply:
We'll track this in https://github.com/kokkos/kokkos-kernels/issues/2611